client/src/routes/Pages/components/Header/Header.scss

```diff
@@ -1,7 +1,23 @@
-@import '../../../../scss/variables';
+@import "../../../../scss/variables";
 
 $header-height: 85px;
 
+.active {
```

Comment:
Can we move this to `.page-header nav`? It will help in preventing conflicts with `active` class name from other components and stylesheets.

Reply:
Absolutely, thanks for the suggestion.